rootfs/docker-entrypoint-initdb.d/003_restore_from_backup.sh

```diff
@@ -50,20 +50,11 @@ EOF
   echo "restore_command = 'envdir /etc/wal-e.d/env wal-e wal-fetch \"%f\" \"%p\"'" >> "$PGDATA/recovery.conf"
   chown -R postgres:postgres "$PGDATA"
   chmod 0700 "$PGDATA"
-  gosu postgres pg_ctl -D "$PGDATA" \
+  su-exec postgres pg_ctl -D "$PGDATA" \
       -o "-c listen_addresses=''" \
       -w start
-
-  echo "Waiting for recovery completion..."
```

Comment:
Can you add a comment or two how this changed? It seems like maybe `postgres` got smarter and maybe provides a lock file for the backup process, or ... — I am not an expert.

Reply:
When the original script was resumed, the push was unified once. In fact, this operation is unnecessary. Because there is another lock in do_backup.

See:
https://github.com/duanhongyi/postgres/blob/master/rootfs/bin/do_backup

Reply:
Sounds good, thanks for the explanation and information.